openapi.yml

```diff
@@ -34,25 +73,60 @@ paths:
       security:
         - bearerAuth: []
       responses:
+        <<: *common-responses
         200:
           description: Successful response
-        401:
-          description: Unouthorized response
-        500:
-          description: Error response
+  /profile:
+    get:
+      security:
+        - bearerAuth: []
+      responses:
+        <<: *common-responses
+        200:
+          description: The user profile information
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/UserProfile"
+    post:
+      security:
+        - bearerAuth: []
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                workEmail:
+                  $ref: "#/components/schemas/EmailAddress"
+              required:
+                - workEmail
+      responses:
+        <<: *common-responses
+        200:
+          description: The updated user profile information
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/UserProfile"
   /public-administrations:
     get:
       parameters:
         - name: search
           in: query
           required: true
-          description: "**Public administration name**. The API responds with a list of results that match the searching words."
+          description: |
+            **Public administration name**.
+            The API responds with a list of results that match the searching words.
           schema:
             type: string
             example: comune gioiosa
       responses:
+        <<: *common-responses
         200:
-          description: The public administrations whose names match the searching words
+          description: |-
+            The public administrations whose names match the searching words
```

Comment:
I'd say: `Public Administrations matching the search criteria.` or something like that.